ui/raidboss/data/07-dt/hunts/living_memory.ts

```diff
@@ -245,14 +332,208 @@ const triggerSet: TriggerSet<Data> = {
         next: Outputs.next,
       },
     },
+
+    // ****** Boss Fate: Mica the Magical Mu ****** //
+    // Aoe (9733 is cast shortly later as the actual aoe)
+    {
+      id: 'Hunt Mica the Magical Mu Spark of Imagination',
+      type: 'StartsUsing',
+      netRegex: { id: '9732', source: 'Mica the Magical Mu', capture: false },
+      response: Responses.aoe(),
+    },
+    // tank buster cleave (part of 9730 Shimmerstrike)
+    {
+      id: 'Hunt Mica the Magical Mu Shimmerstorm Tankbuster',
+      type: 'StartsUsing',
+      netRegex: { id: '9731', source: 'Mica the Magical Mu', capture: true },
+      suppressSeconds: 1,
+      response: Responses.tankCleave(),
+    },
+    // random puddles under people. Shimmerstorm (972F) or Shimmerstrike (9730) spawn them.
+    {
+      id: 'Hunt Mica the Magical Mu Shimmerstorm',
+      type: 'StartsUsing',
+      netRegex: { id: '972F', source: 'Mica the Magical Mu', capture: false },
+      suppressSeconds: 1,
+      response: Responses.moveAway(),
+    },
+    // donut followed by point-blank aoe (972D Twinkling Ring -> 9729 Twinkling Flourish)
+    {
+      id: 'Hunt Mica the Magical Mu Round of Applause',
+      type: 'StartsUsing',
+      netRegex: { id: '972B', source: 'Mica the Magical Mu', capture: false },
+      response: Responses.getInThenOut(),
+    },
+    {
+      id: 'Hunt Mica the Magical Mu Twinkling Ring',
+      type: 'Ability',
+      netRegex: { id: '972D', source: 'Mica the Magical Mu', capture: false },
+      suppressSeconds: 1,
+      response: Responses.getOut(),
+    },
+    // point-blank aoe followed by donut (972A Twinkling Flourish -> 972C Twinkling Ring)
+    {
+      id: 'Hunt Mica the Magical Mu Flourishing Bow',
+      type: 'StartsUsing',
+      netRegex: { id: '9728', source: 'Mica the Magical Mu', capture: false },
+      response: Responses.getOutThenIn(),
+    },
+    {
+      id: 'Hunt Mica the Magical Mu Twinkling Flourish',
+      type: 'Ability',
+      netRegex: { id: '972A', source: 'Mica the Magical Mu', capture: false },
+      suppressSeconds: 1,
+      response: Responses.getIn(),
+    },
+    // protein pizza slices, echo after (9726 4.7s, 9727 6.7)
+    {
+      id: 'Hunt Mica the Magical Mu Double Misdirect',
+      type: 'StartsUsing',
+      netRegex: { id: '9725', source: 'Mica the Magical Mu', capture: false },
+      infoText: (_data, _matches, output) => {
+        return output.protean!();
+      },
+      outputStrings: {
+        protean: 'Protean (x2)',
```

Comment:
This could use a follow-up call for the second dodge, or some other sort of indication that you should be dodging into the first set of Proteans.

Reply:
I did originally want to say something like that. But after looking at other examples where this kind of mechanic happens its never called out.